Services/Repository/RecommendedContent/classes/class.ilDashboardRecommendedContentGUI.php

```diff
@@ -118,14 +118,12 @@ public function confirmedRemoveObject(): void
     {
         $rec_manager = new ilRecommendedContentManager();
         $refIds = (array) ($this->http->request()->getParsedBody()['ref_id'] ?? []);
-        if ($refIds === []) {
-            $this->ctrl->redirect($this, 'manage');
-        }
-
-        foreach ($refIds as $ref_id) {
-            $rec_manager->declineObjectRecommendation($this->user->getId(), (int) $ref_id);
+        if ($refIds !== []) {
+            foreach ($refIds as $ref_id) {
+                $rec_manager->declineObjectRecommendation($this->user->getId(), (int) $ref_id);
+            }
+            $this->main_tpl->setOnScreenMessage('success', $this->lng->txt('pd_remove_multi_confirm'), true);
```

Comment:
Is the PR mixing favourite and recommended content issues on purpose or is this a mistake?

Reply:
removed